splinter/driver/lxmldriver.py

```diff
@@ -266,7 +266,7 @@ def _element_is_link(self, element):
         return element.tag == 'a'
 
     def _element_is_control(self, element):
-        return hasattr(element, 'type')
+        return hasattr(element, 'type') or element.tag == 'textarea'
```

Comment:
Seems like the former should also be checking for `element.tag == 'input'`, but I didn't want to mess with it.